management/user_test.go

```diff
@@ -80,11 +101,34 @@ func TestUser(t *testing.T) {
 		t.Logf("%v\n", uu)
 	})
 
-	t.Run("Delete", func(t *testing.T) {
-		err = m.User.Delete(auth0.StringValue(u.ID))
+	t.Run("GetRoles", func(t *testing.T) {
+		var roles []*Role
+		roles, err = m.User.GetRoles(auth0.StringValue(u.ID))
+		if err != nil {
+			t.Error(err)
+		}
+		t.Logf("%v\n", roles)
+	})
+
+	t.Run("AssignRoles", func(t *testing.T) {
+		roles := []*Role{r1, r2}
+		err = m.User.AssignRoles(auth0.StringValue(u.ID), roles...)
 		if err != nil {
 			t.Error(err)
 		}
+
+	})
+
+	t.Run("UnassignRoles", func(t *testing.T) {
+		roles := []*Role{r1, r2}
+		err = m.User.UnassignRoles(auth0.StringValue(u.ID), roles...)
+		if err != nil {
+			t.Fatal(err)
```

Comment:
Can we make this a `t.Error()` please? Then we're good to merge ;)

Reply:
Sorry, I misunderstood your previous comment... no its fixed.

Reply:
Uhm, did you forget to commit the change? 😄

Reply:
Argh, I have committed the changes to the wrong file (I removed `job_test.go` locally for testing). I have undone these changes now.

Reply:
Ok. In case you want to run tests in isolation, what I usually do is run a specific test case using the `-run` args to `go test`.

For example running the user tests you could do the following:

```
go test -v ./... -run ^TestUser$
```